neo/Prompt/Commands/Wallet.py

```diff
@@ -620,6 +621,23 @@ def command_desc(self):
         return CommandDesc('multisig_addr', 'import a multi-signature address', [p1, p2, p3])
 
 
+class CommandWalletImportToken(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        if len(arguments) != 1:
+            print("Please specify the required parameter")
+            return None
```

Comment:
Is there a reason to specify `return None`? Previously we have just used `return`. See https://github.com/CityOfZion/neo-python/blob/b54648a5552f8ded02b98ddccf95b1a23560f640/neo/Prompt/Commands/Wallet.py#L89-L95 etc.
Similary, I would remove `None` here https://github.com/CityOfZion/neo-python/blob/b54648a5552f8ded02b98ddccf95b1a23560f640/neo/Prompt/Commands/Wallet.py#L286

Reply:
Fixed, thank you 👍